src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/Html/DefaultHtmlGenerator.cs

```diff
@@ -898,11 +907,11 @@ internal static string EvalString(ViewContext viewContext, string key, string fo
         /// <remarks>
         /// Not used directly in HtmlHelper. Exposed for use in DefaultDisplayTemplates.
         /// </remarks>
-        internal static TagBuilder GenerateOption(SelectListItem item, string encodedText, IHtmlEncoder htmlEncoder)
+        internal static TagBuilder GenerateOption(SelectListItem item, IHtmlContent content)
```

Comment:
This method is never called with anything except a `StringHtmlContent` instance. Why not remove the `htmlEncoder` parameter, rename `encodedText` -> `text`, and create the `StringHtmlContent` here?